style.css

```diff
@@ -0,0 +1,73 @@
+html {
+  background: #444444;
+}
+
+body {
+  font-family: tahoma;
+  text-align: center;
+  background: rgb(50, 60, 103);
+  width: 800px;
+  margin: auto;
+  margin-top: 20px;
+  border-radius: 10px;
+  padding-top: 5px;
+  padding-bottom: 20px;
+}
+
+h1 {
+  color: white;
```

Comment:
Colours chosen are comfortable for the eyes and makes the content easy to read on the page